pdc_client/__init__.py

```diff
@@ -90,6 +90,9 @@ def read_config_file(server_alias):
     return result
 
 
+class NoResultsError(Exception):
+    pass
```

Comment:
User should see details about what went wrong in the response (`{"detail":...}`, `{"bad_field":...}` or similar) so it should be accessible from the exception (e.g. `e.response().get('detail')`).

Also add docstrings here and to the `results()` method.

Reply:
oh “so don't stringify it”， I think you don't agree with add a string here.
This makes sense to me

Reply:
Have changed the docstring of results function. Add the example in PDCClient docstring.
And also add string in Exception, not sure if this is what your mean, if not, would you please give me an exactly example?